packages/forms/src/UIForm/UIForm.component.test.js

```diff
@@ -435,6 +426,7 @@ describe('UIForm component', () => {
 			// given
 			const wrapper = mount(
 				<UIFormComponent {...data} {...props} errors={{ firstname: 'firstname is required' }} />,
+				{ attachTo: document.body },
```

Comment:
Need to mount on `document.body` with talend/script jest/enzyme version, or the `document.activeElement` is not set.